common/src/main/java/com/wynntils/features/inventory/CustomBankPagesFeature.java

```diff
@@ -0,0 +1,215 @@
+ */
+package com.wynntils.features.inventory;
+
+import com.wynntils.core.components.Models;
+import com.wynntils.core.config.Category;
+import com.wynntils.core.config.Config;
+import com.wynntils.core.config.ConfigCategory;
+import com.wynntils.core.config.ConfigHolder;
+import com.wynntils.core.config.RegisterConfig;
+import com.wynntils.core.features.Feature;
+import com.wynntils.mc.event.ContainerClickEvent;
+import com.wynntils.mc.event.ContainerCloseEvent;
+import com.wynntils.mc.event.ScreenInitEvent;
+import com.wynntils.mc.event.SlotRenderEvent;
+import com.wynntils.utils.mc.McUtils;
+import com.wynntils.utils.wynn.ContainerUtils;
+import java.util.List;
+import net.minecraft.ChatFormatting;
+import net.minecraft.client.gui.screens.inventory.AbstractContainerScreen;
+import net.minecraft.nbt.CompoundTag;
+import net.minecraft.network.chat.Component;
+import net.minecraft.world.item.ItemStack;
+import net.minecraft.world.item.Items;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+import org.lwjgl.glfw.GLFW;
+
+@ConfigCategory(Category.INVENTORY)
+public class CustomBankPagesFeature extends Feature {
+    // Change to ranged integer when implemented
+    @RegisterConfig
+    public final Config<Integer> buttonOnePage = new Config<>(1);
+
+    @RegisterConfig
+    public final Config<Integer> buttonTwoPage = new Config<>(5);
+
+    @RegisterConfig
+    public final Config<Integer> buttonThreePage = new Config<>(9);
+
+    @RegisterConfig
+    public final Config<Integer> buttonFourPage = new Config<>(13);
+
+    @RegisterConfig
+    public final Config<Integer> buttonFivePage = new Config<>(17);
+
+    @RegisterConfig
+    public final Config<Integer> buttonSixPage = new Config<>(21);
+
+    private static final int MAX_BANK_PAGES = 21;
+    private static final int NEXT_PAGE_SLOT = 8;
+    private static final int PREVIOUS_PAGE_SLOT = 17;
+    private static final List<Integer> BUTTON_SLOTS = List.of(7, 16, 25, 34, 43, 52);
+    private static final List<Integer> QUICK_JUMP_DESTINATIONS = List.of(1, 5, 9, 13, 17, 21);
+
+    private boolean isBankScreen = false;
+    private boolean quickJumping = false;
+    private int currentPage = 1;
+    private int lastPage = MAX_BANK_PAGES;
+    private int pageDestination = 1;
+    private List<Integer> customJumpDestinations;
+
+    @SubscribeEvent
+    public void onScreenInit(ScreenInitEvent e) {
+        if (!(e.getScreen() instanceof AbstractContainerScreen<?> screen)) return;
+        if (!Models.Container.isBankScreen(screen)) return;
+
+        isBankScreen = true;
+
+        currentPage = Models.Container.getCurrentBankPage(screen);
+
+        customJumpDestinations = List.of(
+                buttonOnePage.get(),
+                buttonTwoPage.get(),
+                buttonThreePage.get(),
+                buttonFourPage.get(),
+                buttonFivePage.get(),
+                buttonSixPage.get());
+
+        if (quickJumping && pageDestination > lastPage) {
```

Comment:
Try converting this based on
`if (!quickJumping) return;`

and then simplify the following if statements